components/discount/discountCheckoutScreen.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FunctionComponent } from "react";
 import styles from "../../styles/discount.module.css";
-import RegisterDiscount from "./registerDiscount";
+import RegisterDiscount, { GetCustomCalls } from "./registerDiscount";
 
 type DiscountCheckoutScreenProps = {
```

Comment:
Is this component really useful ? It seems like a wrapper for `RegisterDiscount` 

It would delete a lot of useless code to directly use `RegisterDiscount ` where needed and add ` <div className={styles.container}>` in it 

You have more context than me as I don't remember so well this part of the code. Wdyt ?